docs/guide/front-end-assets.md

```diff
@@ -1,4 +1,4 @@
-# Adding front end CSS and JavaScript
+# Adding front end CSS and JavaScript and other assets
```

Comment:
I don't think this page needs to include this additional note and title change. If you want to have a link from the basic guides to the new page then the Front End Tips page would be more appropriate.

Reply:
"and other assets" was an accidental leftover. Fixed.